core/src/test/java/ai/timefold/solver/core/impl/heuristic/selector/move/MoveSelectorFactoryTest.java

```diff
@@ -201,15 +202,58 @@ void applySorting_withoutAnySortingClass() {
 
     @Test
     void applySorting_withSorterComparatorClass() {
-        final MoveSelector<TestdataSolution> baseMoveSelector = SelectorTestUtils.mockMoveSelector();
-        DummyMoveSelectorConfig moveSelectorConfig = new DummyMoveSelectorConfig();
-        moveSelectorConfig.setSorterOrder(SelectionSorterOrder.ASCENDING);
-        moveSelectorConfig.setSorterComparatorClass(DummyComparator.class);
+        // Old setting
+        {
```

Comment:
Please don't do this.
Different things deserve different test methods, if not even different test classes.
You will appreciate this in a few months' time, when we do Solver 2.0 and you're going to have to remove all the old code.